app/models/profile.rb

```diff
@@ -1,19 +1,8 @@
 class Profile < ActiveRecord::Base
   belongs_to :user, dependent: :destroy
-  has_many :memberships
+  has_many :memberships, dependent: :destroy
```

Comment:
What do you think about backing these relations with foreign keys and cascading deletes? We're on a version of Rails with support for referential integrity and adequate constraints at the db level, seems like we should take advantage of that.